### PR TITLE
[slides] MLP 

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-690/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/7 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/8 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/6 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/9 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/10 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/11 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/12 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/13 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/14 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

**Comment by mli** [comment]
Job d2l-zh/PR-690/16 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-690/

File: chapter_multilayer-perceptrons/mlp.md [hunk 1]
```diff
@@ -153,9 +156,9 @@ $$\operatorname{pReLU}(x) = \max(0, x) + \alpha \min(0, x).$$
 
 ### sigmoid函数
 
-*sigmoid函数*将定义域在$\mathbb{R}$中的输入变换为区间(0, 1)上的输出。因此，sigmoid通常称为*挤压函数*（Squashing function）：它将范围(-inf, inf)中的任意输入压缩到区间(0, 1)中的某个值：
+对于一个定义域在$\mathbb{R}$中的输入，[** *sigmoid函数*将输入变换为区间(0, 1)上的输出**]。因此，sigmoid通常称为*挤压函数*（Squashing function）：它将范围(-inf, inf)中的任意输入压缩到区间(0, 1)中的某个值：
```
Comment (astonzhang) [inline chapter_multilayer-perceptrons/mlp.md:159]: Squashing -> squashing

**Pull request merged**